app/views/candidate_mailer/_interested_without_rejections.erb

```diff
@@ -0,0 +1,9 @@
+# Are you still interested in teaching?
+
+You didn’t pursue your teacher training application, but it’s not too late to apply again for a course if you’ve changed your mind.
+
+We’ve saved your last application, so all you have to do is choose a course, review your application and re-submit.
+
+Sign in to apply again:
+
+https://www.apply-for-teacher-training.education.gov.uk/candidate/account
```

Comment:
These should should the URL helpers, and include a magic link.

Reply:
👍